templates/patterns/featured_media/pattern-featured-media.html.twig

```diff
@@ -24,7 +24,14 @@
       {% set video_alignment_class = 'order-md-1' %}
     {% endif %}
 
-    <h2 class="mb-4">{{ title }}</h2>
+    {%- if title is not empty -%}
+      {% include '@oe-bcl/heading' with {
+        title: title,
+        title_tag: title_tag|default('h2'),
+        title_link: title_link|default(''),
```

Comment:
here we have the choice between an undefined variable and an empty string, does it make sense to set it?

Reply:
I removed the default on title_link.